### PR TITLE
Wonbin

### PR DESCRIPTION
**Pull request by wonbinjeong**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] JavaDocs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Code has been properly commented
- [x] Camel case has been used

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

Issue Number: U.S 2

## What is the new behavior?

- Can add questions
- Can add comments to questions
-

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: code/app/src/main/java/com/cmput301w21t06/crowdfly/Controllers/QuestionContent.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
+package com.cmput301w21t06.crowdfly.Controllers;
+
```
Comment (mu6-Abdullah) [inline code/app/src/main/java/com/cmput301w21t06/crowdfly/Controllers/QuestionContent.java]: small note, for any adapter classes we have went with the naming convention of (ClassName)Adapter, ex. we changed ExperimentContent to ExperimentAdapter

File: code/app/src/main/java/com/cmput301w21t06/crowdfly/Database/CommentController.java [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+    private String eID;
+
+    public CommentController(String eid, String qID) {
+        this.qID = qID;
+        this.eID = eid;
+        cCollection = GodController.getDb().collection(CrowdFlyFirestorePaths.comments(eid, qID));
+        setUp();
+    }
+    /**
+     * This sets up the snapshot listener for trials
```
Comment (mu6-Abdullah) [inline code/app/src/main/java/com/cmput301w21t06/crowdfly/Database/CommentController.java]: do you mean this sets up the snapshot listener for comments?

File: code/app/src/main/java/com/cmput301w21t06/crowdfly/Views/ViewTrialLogActivity.java [hunk 3]
```diff
@@ -175,8 +167,14 @@ public void onClick(View v) {
...
             public void onClick(View v) {
-                Intent intent = new Intent(ViewTrialLogActivity.this, ViewQuestionLogActivity.class);
-                startActivity(intent);
+                if(subscribed || isOwner){
+                    Intent intent = new Intent(ViewTrialLogActivity.this, ViewQuestionLogActivity.class);
+                    intent.putExtra("expID", String.valueOf(expID));
+                    startActivity(intent);
+                }
+                else {
+                    Toaster.makeToast(ViewTrialLogActivity.this, "Please subscribe to the experiment to add trials");
```
Comment (mu6-Abdullah) [inline code/app/src/main/java/com/cmput301w21t06/crowdfly/Views/ViewTrialLogActivity.java:242]: Please subscribe to the experiment to ask questions?

File: code/app/src/main/java/com/cmput301w21t06/crowdfly/Controllers/QuestionContent.java (hunk 1 above)
Comment (wonbinjeong) [reply code/app/src/main/java/com/cmput301w21t06/crowdfly/Controllers/QuestionContent.java]: Weird, that file should be outdated and deleted from my end... I guess Git decided to keep the file for some reason

File: code/app/src/main/java/com/cmput301w21t06/crowdfly/Database/CommentController.java (hunk 2 above)
Comment (wonbinjeong) [reply code/app/src/main/java/com/cmput301w21t06/crowdfly/Database/CommentController.java]: I fixed these comments and javadocs, i'll be posting a pr for this soon!

**Review by mu6-Abdullah — COMMENTED** [review]
I made a couple comments, and mainly throughout the code the only issue is related to comments. A couple of comments say trials, when I think they should say questions, or comments

**Review by mu6-Abdullah — COMMENTED** [review]
Can we also change so userID displays the user's displayID, not their actual firebase id?

**Review by mu6-Abdullah — APPROVED** [review]

**Pull request merged**